SwiftLint/swiftlint+frameworks.yml

```diff
@@ -7,3 +7,7 @@
 
 #opt_in_rules:
 #  - explicit_top_level_acl
+#  - missing_docs
+
+#missing_docs:
+#  warning: open, public, internal
```

Comment:
I'm afraid that `internal` will flood us with warnings. Maybe we could go step by step and enable only `public` and `open` as a first step, and prepare the field for `internal` before enabling it.